.circleci/config.yml

```diff
@@ -397,7 +402,7 @@ workflows:
           type: approval
           requires:
             - deploy-to-staging
-      - assume-role-production:
+      - assume-role-production-1:
```

Comment:
rather than creating a dupe definition of the job with a different name
instead make use of 'name' key right here to give the step itself a name explicitly.
see how to do that in CCI docs